flambe/runner/run.py

```diff
@@ -114,6 +119,8 @@ def main(args: argparse.Namespace) -> None:
                              'when using this flag as it could have undesired effects.')
     parser.add_argument('-s', '--secrets',
                         type=str, default=os.path.join(FLAMBE_GLOBAL_FOLDER, "secrets.ini"))
+    parser.add_argument('-d', '--debug', action='store_true',
+                        help='Enable debug mode, overriding config.yaml setting')
```

Comment:
This no longer overrides `config.yaml` setting, right? Also I think we should explain a bit more what debug does, e.g. explicitly say "For an experiment job, Ray will run in a single thread if debug mode is enabled."

Reply:
Done.